src/api/v2/v2AzureResourcesApi.ts

```diff
@@ -200,7 +200,7 @@ export interface V2AzureResourcesApi extends AzureResourcesApiBase {
      * @param id The resolver ID. Must be unique.
      * @param resolver The resolver
      */
-    registerApplicationResourceBranchDataProvider<T extends ResourceModelBase>(id: string, provider: BranchDataProvider<ApplicationResource, T>): vscode.Disposable;
+    registerApplicationResourceBranchDataProvider<T extends ResourceModelBase>(id: AzExtResourceType, provider: BranchDataProvider<ApplicationResource, T>): vscode.Disposable;
```

Comment:
Arg is still named `id` here and in [`v2AzureResourcesApiWrapper.ts`](https://github.com/microsoft/vscode-azureresourcegroups/pull/381/commits/2173759510366be4f3ae771e3e9582585798efbe#diff-960368edff109f85fc750a3ca6cd7df9d3e64c40d40221aae758490a3156526b)